command/ripe_atlas/credits.go

```diff
@@ -0,0 +1,72 @@
+package ripe_atlas
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/innogames/slack-bot/v2/bot"
+	"github.com/innogames/slack-bot/v2/bot/matcher"
+	"github.com/innogames/slack-bot/v2/bot/msg"
+	"github.com/innogames/slack-bot/v2/client"
+	"github.com/pkg/errors"
+	"net/http"
+)
+
+type creditsCommand struct {
+	bot.BaseCommand
+	cfg Config
+}
+
+func (c *creditsCommand) GetMatcher() matcher.Matcher {
+	return matcher.NewGroupMatcher(
+		matcher.NewTextMatcher("credits", c.credits),
+	)
+}
+
+func (c *creditsCommand) credits(match matcher.Result, message msg.Message) {
+
+	c.AddReaction(":coffee:", message)
+	defer c.RemoveReaction(":coffee:", message)
+
+	url := fmt.Sprintf("%s/credits", c.cfg.APIURL)
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		c.ReplyError(message, errors.Wrap(err, "Request creation returned an err"))
```

Comment:
We might want to log these kind of errors somewhere. Not sure how it's done in this project though.